### PR TITLE
#3883 Update inspect command

### PR DESCRIPTION
**Pull request by uilianries**
Related Issue: https://github.com/conan-io/conan/pull/3914

### CODE REVIEW

File: getting_started.rst [hunk 1]
```diff
@@ -81,6 +81,60 @@ An MD5 Encrypter using the Poco Libraries
         short_paths: False
         apply_env: True
         build_policy: None
+        settings: ('os', 'arch', 'compiler', 'build_type')
+        options
+            cxx_14: [True, False]
+            enable_apacheconnector: [True, False]
+            enable_cppparser: [True, False]
```
Comment (danimtb) [inline getting_started.rst]: I think this output is too much for the getting started section.
Please list the fist two options and default options and write `...` below them:

```
    options:
        shared: [True, False]
        enable_xml: [True, False]
        ...
    default_options: 
        shared=False
        enable_xml=True
        ...
```

Comment (danimtb) [reply getting_started.rst]: Why are options and default options in different order?

File: reference/commands/misc/inspect.rst [hunk 2]
```diff
@@ -68,3 +68,7 @@ If no specific attributes are defined via ``-a``, then, some default attributes
     apply_env: True
     build_policy: None
     topics: None
+    settings: ('os', 'arch', 'compiler', 'build_type')
+    options
```
Comment (danimtb) [inline reference/commands/misc/inspect.rst]: Why it is not `options:` ?

File: reference/commands/misc/inspect.rst [hunk 3]
```diff
@@ -68,3 +68,7 @@ If no specific attributes are defined via ``-a``, then, some default attributes
...
+        shared: [True, False]
+    default_options: shared=False
```
Comment (danimtb) [inline reference/commands/misc/inspect.rst]: I would like to see this in the same shape of the options section

File: getting_started.rst (hunk 1 above)
Comment (uilianries) [reply getting_started.rst]: I extracted from command output, probably is related to declared order.

File: reference/commands/misc/inspect.rst (hunk 2 above)
Comment (uilianries) [reply reference/commands/misc/inspect.rst]: I need to check inspect output, but your argument is authentic.

File: reference/commands/misc/inspect.rst (hunk 3 above)
Comment (uilianries) [reply reference/commands/misc/inspect.rst]: I got from command inspect result. I could update, but won't reflect the real output.

Comment (uilianries) [reply reference/commands/misc/inspect.rst]: Done!

File: reference/commands/misc/inspect.rst (hunk 2 above)
Comment (uilianries) [reply reference/commands/misc/inspect.rst]: Done!

File: getting_started.rst (hunk 1 above)
Comment (uilianries) [reply getting_started.rst]: Done!

File: getting_started.rst [hunk 4]
```diff
@@ -81,6 +81,14 @@ An MD5 Encrypter using the Poco Libraries
         short_paths: False
         apply_env: True
         build_policy: None
+        settings: ('os', 'arch', 'compiler', 'build_type')
+        options:
+            enable_apacheconnector: [True, False]
+            shared: [True, False]
+        default_options:
+            enable_apacheconnector=False
```
Comment (danimtb) [inline getting_started.rst]:
```suggestion
            enable_apacheconnector: False
```

File: getting_started.rst [hunk 5]
```diff
@@ -81,6 +81,14 @@ An MD5 Encrypter using the Poco Libraries
...
+            shared=False
```
Comment (danimtb) [inline getting_started.rst]:
```suggestion
            shared: False
```

File: reference/commands/misc/inspect.rst [hunk 6]
```diff
@@ -68,3 +68,8 @@ If no specific attributes are defined via ``-a``, then, some default attributes
     apply_env: True
     build_policy: None
     topics: None
+    settings: ('os', 'arch', 'compiler', 'build_type')
+    options:
+        shared: [True, False]
+    default_options:
+        shared=False
```
Comment (danimtb) [inline reference/commands/misc/inspect.rst]:
```suggestion
        shared: False
```

Comment (uilianries) [reply reference/commands/misc/inspect.rst]: done!

File: getting_started.rst (hunk 5 above)
Comment (uilianries) [reply getting_started.rst]: done!

File: getting_started.rst (hunk 4 above)
Comment (uilianries) [reply getting_started.rst]: done!

**Pull request merged**